membership.yml

```diff
@@ -42,6 +42,7 @@ organization:
       - srekapalli
       - stuart-pollock
       - xavileon
+      - lorin
```

Comment:
Mind adding this in alphabetical order so it's easier to find people in the list?  Thanks!

Reply:
My bad. Will do!